packages/next-tinacms-cloudinary/src/handlers.ts

```diff
@@ -164,18 +175,34 @@ async function deleteAsset(req: NextApiRequest, res: NextApiResponse) {
     })
   })
 }
+function getCloudinaryToTinaFunc(opts: CloudinaryOptions) {
+  return function cloudinaryToTina(file: any): Media {
+    // TODO: I want to use this but it seams we might have to update our webpack config in order to do this in node
+    // const useHttps = opts?.useHttps ?? true
+
+    // Default to true
+    let useHttps = true
+    if (typeof opts !== 'undefined' && typeof opts.useHttps !== 'undefined') {
+      useHttps = opts.useHttps
+    }
 
-function cloudinaryToTina(file: any): Media {
-  const filename = path.basename(file.public_id)
-  const directory = path.dirname(file.public_id)
-
-  return {
-    id: file.public_id,
-    filename,
-    directory,
-    src: file.url,
-    previewSrc: transformCloudinaryImage(file.url, 'w_75,h_75,c_fill,q_auto'),
-    type: 'file',
+    const sel = useHttps ? ('secure_url' as const) : ('url' as const)
+    console.log({ sel })
```

Comment:
Look goods 👍 Let's remove this `console.log()` here, though.

Reply:
Good catch!